src/components/Avatar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.scss';
+
+const Avatar = props => (
+  <div className={`avatar ${props.className}`}>
```

Comment:
Can we go with wds-avatar instead of making our own? https://fandomdesignsystem.com/#/components/avatars

Reply:
Yeah, interesting. I copied this from our repo. I'm going to talk with derek to make sure we are consistent on the design and then I will make the changes